derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -351,21 +353,23 @@ impl Field {
             ..
         } = &self
         {
-            return Err(darling::Error::unsupported_format(
-                r#"#[builder(default)] and #[builder(build="...")] cannot be used together"#,
-            )
-            .with_span(field_default));
+            errors.push(
+                darling::Error::unsupported_format(
```

Comment:
```suggestion
                darling::Error::custom(
```

The docs for `unsupported_format` imply it's narrowly-applicable to the case of meta-items that are lists vs name-value pairs. Here, `custom` will be better.